isis/tests/FunctionalTestsIsis2pds.cpp

```diff
@@ -0,0 +1,435 @@
+  EXPECT_EQ(int(imageObject["BANDS"]), 1);
+  EXPECT_EQ(QString(imageObject["BAND_STORAGE_TYPE"]), "BAND_SEQUENTIAL");
+  EXPECT_DOUBLE_EQ(double(imageObject["OFFSET"]), 0.0);
+  EXPECT_DOUBLE_EQ(double(imageObject["SCALING_FACTOR"]), 1.0);
+  EXPECT_EQ(int(imageObject["SAMPLE_BITS"]), 32);
+  EXPECT_EQ(QString(imageObject["SAMPLE_BIT_MASK"]), "2#11111111111111111111111111111111#");
+  EXPECT_EQ(QString(imageObject["SAMPLE_TYPE"]), "PC_REAL");
+  EXPECT_EQ(QString(imageObject["CORE_NULL"]), "16#FF7FFFFB#");
+  EXPECT_EQ(QString(imageObject["CORE_LOW_REPR_SATURATION"]), "16#FF7FFFFC#");
+  EXPECT_EQ(QString(imageObject["CORE_LOW_INSTR_SATURATION"]), "16#FF7FFFFD#");
+  EXPECT_EQ(QString(imageObject["CORE_HIGH_REPR_SATURATION"]), "16#FF7FFFFF#");
+  EXPECT_EQ(QString(imageObject["CORE_HIGH_INSTR_SATURATION"]), "16#FF7FFFFE#");
+
+  PvlObject projectionObject = outputLabel.findObject("IMAGE_MAP_PROJECTION");
+
+  EXPECT_EQ(QString(projectionObject["MAP_PROJECTION_TYPE"]), "SIMPLE CYLINDRICAL");
+  EXPECT_EQ(QString(projectionObject["PROJECTION_LATITUDE_TYPE"]), "PLANETOCENTRIC");
+  EXPECT_DOUBLE_EQ(double(projectionObject["A_AXIS_RADIUS"]), 1737.4);
+  EXPECT_DOUBLE_EQ(double(projectionObject["B_AXIS_RADIUS"]), 1737.4);
+  EXPECT_DOUBLE_EQ(double(projectionObject["C_AXIS_RADIUS"]), 1737.4);
+  EXPECT_EQ(QString(projectionObject["FIRST_STANDARD_PARALLEL"]), "N/A");
+  EXPECT_EQ(QString(projectionObject["SECOND_STANDARD_PARALLEL"]), "N/A");
+  EXPECT_EQ(QString(projectionObject["COORDINATE_SYSTEM_NAME"]), "PLANETOCENTRIC");
+  EXPECT_EQ(QString(projectionObject["POSITIVE_LONGITUDE_DIRECTION"]), "EAST");
+  EXPECT_EQ(QString(projectionObject["KEYWORD_LATITUDE_TYPE"]), "PLANETOCENTRIC");
+  EXPECT_DOUBLE_EQ(double(projectionObject["CENTER_LATITUDE"]), 0.0);
+  EXPECT_DOUBLE_EQ(double(projectionObject["CENTER_LONGITUDE"]), 180.0);
+  EXPECT_EQ(int(projectionObject["LINE_FIRST_PIXEL"]), 1);
+  EXPECT_EQ(int(projectionObject["LINE_LAST_PIXEL"]), 240);
+  EXPECT_EQ(int(projectionObject["SAMPLE_FIRST_PIXEL"]), 1);
+  EXPECT_EQ(int(projectionObject["SAMPLE_LAST_PIXEL"]), 480);
+  EXPECT_DOUBLE_EQ(double(projectionObject["MAP_PROJECTION_ROTATION"]), 0.0);
+  EXPECT_DOUBLE_EQ(double(projectionObject["MAP_RESOLUTION"]), 1.3333333333333);
+  EXPECT_DOUBLE_EQ(double(projectionObject["MAP_SCALE"]), 22.74252);
+  EXPECT_DOUBLE_EQ(double(projectionObject["MAXIMUM_LATITUDE"]), 90.0);
+  EXPECT_DOUBLE_EQ(double(projectionObject["MINIMUM_LATITUDE"]), -90.0);
+  EXPECT_DOUBLE_EQ(double(projectionObject["EASTERNMOST_LONGITUDE"]), 360.0);
+  EXPECT_DOUBLE_EQ(double(projectionObject["WESTERNMOST_LONGITUDE"]), 0.0);
+  EXPECT_DOUBLE_EQ(double(projectionObject["LINE_PROJECTION_OFFSET"]), 119.5);
+  EXPECT_DOUBLE_EQ(double(projectionObject["SAMPLE_PROJECTION_OFFSET"]), 239.5);
+
+}
+
+TEST(isis2pdsTest, FunctionalTestIsis2pdsPds4) {
+  QTemporaryDir prefix;
+  QString outputFile = prefix.path() + "/outTemp.img";
+  QVector<QString> args = {"from=data/isis2pds/dawnEqui1.cub",
```

Comment:
Same question as above here.

Reply:
This one I could probably use a fixture cube instead. Good call.

Reply:
Ah. I remember the reason I switched to the cub file. When using the fixture, It gives me an error "Unable to find a target in input cube". When I reverted to the cub file, it no longer gave me the error.